libflux/src/flux/semantic/fresh.rs

```diff
@@ -67,6 +67,22 @@ impl<T: Hash + Ord + Eq + Fresh, S> Fresh for HashMap<T, S> {
     }
 }
 
+impl<T: Fresh> Fresh for BTreeMap<String, T> {
```

Comment:
Would be great to get a brief description on the purpose of this ... been seeing this around and still unclear on it.